core/src/main/java/org/pentaho/di/osgi/service/notifier/DelayedInstanceNotifier.java

```diff
@@ -63,7 +63,7 @@ public void run() {
     try {
       factory = osgiPluginTracker.findOrCreateBeanFactoryFor( serviceObject );
     } catch ( OSGIPluginTrackerException e ) {
-      logger.error( "Error getting bean factory", e );
+      logger.warn( "Error getting bean factory. If this error does not self-correct, there may be other startup problems.", e );
```

Comment:
As the log message indicates, this error is self-correcting; the scheduler below will retry this operation until successful or interrupted by a different exception or other condition.  Changing it to a warning to avoid concern since this looks like it can happen during an otherwise successful startup.